src/server/module_processing.cpp

```diff
@@ -81,6 +83,19 @@ namespace {
   }
 }
 
+namespace BridgeApiSV {
```

Comment:
I thought it might be a good idea to have a clear differentiation between the server (SV) and the client (CL) variables and functions. Hence the two namespaces. I've now put everything into the remix_api namespace